extension/cart/addCartItems.js

```diff
@@ -45,8 +45,11 @@ function addItemsToCart (request, accessToken, items, cartId, cartUrl, log, reje
   request.post(options, (err, res) => {
     if (err) return cb(err)
     if (res.statusCode !== 200) {
-      log.error(`Got ${res.statusCode} from magento: ${JSON.stringify(res.body)}`)
-
+      log.warn(
+        {
+          statusCode: res.statusCode,
+          responseBody: JSON.stringify(res.body)
```

Comment:
I know you were just moving out the existing code, but I think it would also make sense to extract the error here like in other places. Since this is a mage error, we expect the same format.